### ISSUE TITLE
Formatting bug deletes incomplete typedef declaration parameter block

### ISSUE THREAD

**Issue by cambiata** [vshaxe/vshaxe#290]
If you forget to use the "=" when declaring a typedef, for example...
```
typedef Testtype {  // <-   "=" assignment is forgotten!
    something: String
}
```
the whole parameter block is deleted by the formatter.  It just becomes:
```
typedef Testtype  // complete declaration block disappeared... :-(
```

BTW, thank you guys for a great job! :-)

**Comment by nadako** [comment]
Submitted to https://github.com/HaxeCheckstyle/haxe-formatter/issues/228, but I guess will leave this issue open until we upgrade the formatter (or what's our preferred policy here, @Gama11?)

**Comment by AlexHaxe** [comment]
fixed in dev

**Comment by Gama11** [comment]
Fixed in vshaxe 2.4.1 since we updated haxe-formatter to 1.1.0.
